clang/unittests/Format/FormatTestCSharp.cpp

```diff
@@ -1304,6 +1304,18 @@ TEST_F(FormatTestCSharp, CSharpGenericTypeConstraints) {
                "}",
                Style);
 
+  // When the where line is not to be formatted, following lines should not take
+  // on its indentation.
+  verifyFormat("class ItemFactory<T>\n"
```

Comment:
Is there no variant which only takes one string, or do I miss the difference between the to strings?

Reply:
There is no variant which only takes one string.